docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -41,7 +41,8 @@ dnf --assumeyes install make gcc gcc-c++ glibc-devel git-core libsecret-devel rp
 #### Arch
 
 ```sh
-TODO: Arch instructions
+# Install the development packges
+pacman -S build-essential libxkbfile-dev libsecret-1-dev libx11-dev
```

Comment:
The packages are not called like that. I will try to find the correct ones.

Reply:
That's a good point, it looks like these were just copied and pasted from Ubuntu? Doesn't arch use `base-devel`. 

@CatPerson136 Did you actually test these instructions yourself on a system?

Reply:
I now added a command that actually works.
I haven't finished a pulsar build though

Reply:
> That's a good point, it looks like these were just copied and pasted from Ubuntu? Doesn't arch use `base-devel`. 
> 
> @CatPerson136 Did you actually test these instructions yourself on a system?

 Thanks for catching that I forgot I put the wrong command by accident

Reply:
> The packages are not called like that.
> 
> I think this should be correct:
> ```suggestion
> pacman -S base-devel libxkbfile libsecret-1 libx11
> ```

 Alright I updated it

Reply:
Did you test it?

```
[dae@archlinux pulsar]$ sudo pacman -S base-devel libxkbfile libsecret-1 libx11
...
error: target not found: libsecret-1
```

We can't keep doing this, you committed the suggested changes and yet if I run these commands it doesn't work - the package on Arch is `libsecret`, not `libsecret-1` (https://archlinux.org/packages/core/x86_64/libsecret/). How can we trust the accuracy of this PR if the author doesn't appear to have run it? The suggestion to change it may not have come from you but you still have to be responsible for checking that change is correct.

I don't mean to be harsh but this is important to get correct and why I spent ages testing and re-testing the original commands I wrote in this doc to ensure they would work on the most base level distributions they apply to. I've installed a minimal Arch system with `archinstall` and running `pacman -S base-devel libxkbfile libsecret-1 libx11` simply does not work.

Reply:
Sorry. That was me writing the message on a different device than using the commands.

Reply:
> Did you test it?
> 
> ```
> [dae@archlinux pulsar]$ sudo pacman -S base-devel libxkbfile libsecret-1 libx11
> ...
> error: target not found: libsecret-1
> ```
> 
> We can't keep doing this, you committed the suggested changes and yet if I run these commands it doesn't work - the package on Arch is `libsecret`, not `libsecret-1` (https://archlinux.org/packages/core/x86_64/libsecret/). How can we trust the accuracy of this PR if the author doesn't appear to have run it? The suggestion to change it may not have come from you but you still have to be responsible for checking that change is correct.
> 
> I don't mean to be harsh but this is important to get correct and why I spent ages testing and re-testing the original commands I wrote in this doc to ensure they would work on the most base level distributions they apply to. I've installed a minimal Arch system with `archinstall` and running `pacman -S base-devel libxkbfile libsecret-1 libx11` simply does not work.

ok done